package.json

```diff
@@ -46,7 +47,7 @@
     "prettier": "2.0.5",
     "rimraf": "3.0.2",
     "rollup": "^2.11.2",
-    "rollup-plugin-terser": "^6.1.0",
+    "rollup-plugin-terser": "^7.0.2",
```

Comment:
Bumped into this issue: https://github.com/terser/terser/issues/903
this version fixes it.